src/partials/scanning-page/button.html

```diff
@@ -0,0 +1 @@
+<a class="order-button" href="/src/order.html">ЗРОБИТИ ЗАМОВЛЕННЯ</a>
```

Comment:
```suggestion
<a class="order-button" href="./order.html">ЗРОБИТИ ЗАМОВЛЕННЯ</a>
```
у нас путь относительный, в проекта src папки не будет она нужна для сборщика)